work-process/user-stories.md

```diff
@@ -9,6 +9,14 @@
   *  they assign people from whom they want to receive feedback for better definition of the User Story.
   * Once these people post their feedback, they unassign themselves from the User Story.
 - Once the Scrum Master and Product Owner agree that the User Story is well defined, they add ``✋ [vote]`` prefix to the title of the User Story. At that point, Scrum Master and/or Product Owner assign people from whom they want to receive User Story Points estimation. With this, online poker planning begins.
+-  Assignees add their estimates in a comment with the following structure:
+```
+<details>
+  <summary>Click to see estimate</summary>
+  SP: (1, 2, 3, 5, 8, 13, 20, 100)
```

Comment:
Remove the `SP:` and use angle brackets to denote options:

<1, 2, 3, 5, 8, 13, 20, 100>

Reply:
Also missing 40

Reply:
Probably best to reference the values in the estimates dropdown rather than specific values here.

Reply:
We should have it 20 and not 21. I see that in zenhub plugin there is 21 but this was because we have not change it.

> use angle brackets to denote options:

You think Niteans will be confused if we keep `SP:` and use regular brackets?

Reply:
So there is less typing